services/natural_language_understanding/v1.js

```diff
@@ -15,8 +15,9 @@
  **/
 
 module.exports = function (RED) {
-  const SERVICE_IDENTIFIER = 'natural-language-understanding';
-  const NaturalLanguageUnderstandingV1 = require('watson-developer-cloud/natural-language-understanding/v1');
+  const SERVICE_IDENTIFIER = 'natural-language-understanding',
+    NaturalLanguageUnderstandingV1 = require('ibm-watson/natural-language-understanding/v1'),
+    { IamAuthenticator } = require('ibm-watson/auth');
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Expected indentation of 8 spaces but found 4.](https://app.codacy.com/manual/BetaWorks-NodeRED-Watson/node-red-node-watson/pullRequest?prid=4804371)